# Respeaker.cfg

#!/usr/bin/env python
#
# WARNING!!
# This file is automatically generated.
# DO NOT MODIFY THIS FILE!
#
# To generate this file, please run:
# cd /path/to/this/package
# python ./scripts/respeaker_gencfg.py
#

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Add parameters for the device settings
gen.add("STATNOISEONOFF", bool_t, 0, "Stationary noise suppression. 0 = OFF 1 = ON", True)
gen.add("GAMMA_ETAIL", double_t, 0, "Over-subtraction factor of echo (tail components). min .. max attenuation", 1.000000, 0.000000, 3.000000)
gen.add("AGCTIME", double_t, 0, "Ramps-up / down time-constant in seconds.", 0.982401, 0.100000, 1.000000)
# 여기에 다른 파라미터들을 추가

exit(gen.generate("respeaker_ros2", "respeaker_ros2", "Respeaker"))
